### ISSUE TITLE
Show console log on the UI

### ISSUE THREAD

**Issue by synogen** [synogen/ftlautosave#6]
If possible switchable so you can close/open the log output.
The command file used for startup right now can be removed once this is done.

**Comment by synogen** [comment]
Low priority as it proved to be even more difficult and messy than I thought due to my complete lack of experience with JavaFX.
For now the command file will have to do for this and once the [indicators for the filewatches](https://github.com/synogen/ftlautosave/issues/12) are implemented there should be little need to watch the log file.

**Comment by synogen** [comment]
Replaced by [this issue](https://github.com/synogen/ftlautosave/issues/13)
